test/jndi_mysql_test.rb

```diff
@@ -24,26 +24,22 @@ def setup
 
     def teardown; ActiveRecord::Base.connection_pool.disconnect! end
 
-    test "configures once" do
+    test "configures connection" do
       pool = ActiveRecord::Base.connection_pool
-      assert_false pool.active_connection? if pool.respond_to?(:active_connection?)
+      assert_nil pool.active_connection? if pool.respond_to?(:active_connection?)
 
-      adapter_class.any_instance.expects(:configure_connection).once
+      # called twice since there's a version check during initialize that needs a connection
+      adapter_class.any_instance.expects(:configure_connection).at_least_once
```

Comment:
this is a bit gross ... and smells but we should have a special configuration to skip configure_connection for good.

Reply:
agreed it's ugly...I think I try the version check dummy thing I mentioned above...

Reply:
untouched in the new version